cli/src/pcluster/config/cluster_config.py

```diff
@@ -1246,6 +1246,7 @@ def __init__(
         self,
         name: str,
         instance_type: str,
+        local_storage: LocalStorage = None,
```

Comment:
Here we should not expect to receive an object from the Marshmallow (schema)

Reply:
Done!